src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -151,39 +163,39 @@ const getDateFormat = match<MarketLabel, string>([
 ])
 
 const DateForm: React.FC<{
-  quoteCartId: string
-  quote: OfferQuote
-  offerData: OfferData
-  isSingleStartDateBundle?: boolean
+  startDate: any
```

Comment:
Shouldn't be `Date | null`?

Reply:
This is because it's a scalar in the schema so it's typed as `any`. Not a great solution but I think a good enough trade-off for now. What do you think?